File: 2018/pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float change;
    do
    {
        change = get_float("Change owed: ");
    }
    while (change < 0);

    change = round(change * 100);
    int cents = (int) change;
    int coins = 0;

    coins = coins + (cents / 25);
    cents = cents % 25;
    coins = coins + (cents / 10);
    cents = cents % 10;
    coins = coins + (cents / 5);
    cents = cents % 5;
    coins = coins + cents;

    printf("%i", coins);
    printf("\n");
}

File: 2018/pset6/cash/cash.py
from cs50 import get_float

while True:
    change = get_float("Change owed: ")
    if change >= 0:
        break

cents = round(change * 100)
coins = 0

coins += cents // 25
cents %= 25
coins += cents // 10
cents %= 10
coins += cents // 5
cents %= 5
coins += cents

print(coins)